poshc2/server/database/DBPostgres.py

```diff
@@ -192,49 +192,6 @@ def new_task(task, user, randomuri):
     conn.commit()
 
 
-def get_lastcommand():
```

Comment:
Nice! This mean the history table can go altogether? I.e. not be created etc either?